docs/installation-and-operation/configuring-application-database.md

```diff
@@ -6,71 +6,129 @@ redirect_from:
+
+The minimum recommended version is MySQL 8.0.17 or MariaDB 10.2.2. The `utf8mb4` character set is required.
+
+You can change the application database to use MySQL using environment variables like so:
+
+```sh
+export MB_DB_TYPE=mysql
+export MB_DB_DBNAME=metabase
+export MB_DB_PORT=3306
+export MB_DB_USER=<username>
+export MB_DB_PASS=<password>
+export MB_DB_HOST=localhost
+java -jar metabase.jar
+```
+
+Metabase won't create this database for you. Example SQL statement to create the database:
+
+```sh
+CREATE DATABASE metabase CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+```
+
+The following command will tell Metabase to look for its application database using the supplied MySQL connection information. Metabase also supports providing a full JDBC connection string if you have additional parameters:
+
+```sh
+export MB_DB_CONNECTION_URI="jdbc:mysql://localhost:3306/metabase?user=<username>&password=<password>"
+java -jar metabase.jar
+```
 
-    metabase.db.h2.db  # Or metabase.db.mv.db depending on when you first started using Metabase.
-    metabase.db.trace.db
+As with Postgres, `MB_DB_CONNECTION_URI` can also be used in combination with `MB_DB_USER` and/or `MB_DB_PASS` if you
+want to pass one or both separately from the rest of the JDBC connection string:
+
+```sh
+export MB_DB_CONNECTION_URI="jdbc:mysql://localhost:5432/metabase"
+export MB_DB_USER=<username>
+export MB_DB_PASS=<password>
+java -jar metabase.jar
+```
 
-If for any reason you want to use an H2 database file in a separate location from where you launch Metabase you can do so using an environment variable. For example:
+## H2(default)
 
-    export MB_DB_TYPE=h2
-    export MB_DB_FILE=/the/path/to/my/h2.db
-    java -jar metabase.jar
+> **For production installations of Metabase we recommend that people [replace the default H2 database with PostgreSQL](./migrating-from-h2.md)**. Postgres offers a greater degree of performance and reliability.
 
-Note that H2 automatically appends `.mv.db` or `.h2.db` to the path you specify; do not include those in your path! In other words, `MB_DB_FILE` should be something like `/path/to/metabase.db`, rather than something like `/path/to/metabase.db.mv.db` (even though this is the file that actually gets created).
+By default, Metabase ships with an [H2 database](https://www.h2database.com/) to make it easy to demo Metabase on your local machine. **Avoid using this default database in production**.
 
-## [Postgres](https://www.postgresql.org/)
+If when launching Metabase you don't provide environment variables that specify connection details for a production database, Metabase will attempt to create a new H2 database in the same directory as the Metabase JAR.
 
-You can change the application database to use Postgres using a few simple environment variables. For example:
+You can see these database files from the terminal:
```

Comment:
```suggestion
You can see the database files for the H2 database from the terminal:
```
The previous paragraph doesn't mention any files so "these files" is an unclear reference, especially considering we never say that H2 is a file database